templates/records/record_disambiguation_page.html

```diff
@@ -25,7 +25,8 @@ <h2 class="sr-only">Associated records</h2>
                 id="analytics-associated-records">
 
                 {% for record in record_results_page %}
-                    {% include 'includes/card-group-record-summary-no-image.html' %}
+                    {# passing empty string as this template does not share description_override  #}
```

Comment:
I think the better fix might actually be to replace the `{% firstof %}` in the other template with a simple if / else - which would prevent this cropping up elsewhere again unexpectedly in future